File: iOS/iOS开发指南(从Hello World到App Store上架)/swift/11/11.4.2/MyNotes/Note.m
//
//  Note.m
//  MyNotes
//
//  Created by andyron on 2016/12/20.
//  Copyright © 2016年 andyron. All rights reserved.
//

#import "Note.h"

@implementation Note

-(instancetype)initWithDate:(NSDate *)date content:(NSString *)content {
    self = [super init];
    
    if (self) {
        self.date = date;
        self.content = content;
    }
    return self;
}

-(instancetype)init {
    self = [super init];
    
    if (self) {
        self.date = [[NSDate alloc] init];
        self.content = @"";
    }
    return self;
}
@end

File: iOS/old/iOS开发指南(从Hello World到App Store上架)/swift/11/11.4.1/MyNotes/Note.swift
//
//  Note.swift
//  MyNotes
//
//  Created by andyron on 2016/12/19.
//  Copyright © 2016年 andyron. All rights reserved.
//

import Foundation

class Note {
    var date: Date
    var content: String
    
    init(date:Date, content:String) {
        self.date = date
        self.content = content
    }
    init() {
        self.date = Date()
        self.content = ""
    }
}
